### PR TITLE
AURA-0 Adding test file story

### PR DESCRIPTION
**Pull request by Antonyacosta78**
TODO: comment code
TODO: Write tests

### CODE REVIEW

**Review by Antonyacosta78 — COMMENTED** [review]
lgtm

**Review by ArukouFX — DISMISSED** [review]
LGTM.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.12.1
```
Comment (ArukouFX) [inline .npmrc]: No needed because the correct file is nvmrc.

Comment (Antonyacosta78) [reply .npmrc:1]: AAAAAa

Comment (Antonyacosta78) [reply .npmrc]: Working on it!

**Review by ArukouFX — APPROVED** [review]
LGTM.

**Pull request merged**